pra_request_tracker/record_requests/tests/factories.py

```diff
@@ -0,0 +1,30 @@
+from factory import Faker
+import factory
+from factory.django import DjangoModelFactory
```

Comment:
Well.....good to know about this :joy: I use faker in another project but was unaware!